### PR TITLE
Add policy for node auto repair

### PR DESCRIPTION
**Pull request by charliewolf**

### CODE REVIEW

File: validator/test/fixtures/gke_node_auto_repair/constraints/enable_auto_repair/data.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: GCPGKENodeAutoRepairConstraintV1
+metadata:
+  name: enable_auto_repair
+  annotations:
+    description: Ensure automatic node repair is enabled on all node pools in a GKE cluster
+    benchmark: CIS11_7.07
```
Comment (joecheuk) [inline validator/test/fixtures/gke_node_auto_repair/constraints/enable_auto_repair/data.yaml]: The benchmark annotation style has been updated, can you also update this and the rest (& previous) accordingly?

Example of the new style:
https://github.com/forseti-security/policy-library/pull/137/files

Comment (charliewolf) [reply validator/test/fixtures/gke_node_auto_repair/constraints/enable_auto_repair/data.yaml]: @joecheuk updated this on my open PRs. by "& previous" do you mean my previously merged policies or do you mean all of the policies currently in the repo?

Comment (joecheuk) [reply validator/test/fixtures/gke_node_auto_repair/constraints/enable_auto_repair/data.yaml]: I meant the policies you authored.

Comment (joecheuk) [reply validator/test/fixtures/gke_node_auto_repair/constraints/enable_auto_repair/data.yaml]: Just to be sure, can you also add in the comment
```
# This constraint is not certified by CIS.
```
That's a requirement from the CIS side.

**Comment by AdrienWalkowiak** [comment]
@joecheuk any other comments for this one?

**Comment by joecheuk** [comment]
@AdrienWalkowiak it's good to go, merging in.

**Pull request merged**